client/test_commands/scenario.py

```diff
@@ -234,11 +236,11 @@ def compute_h_sig_attack_nf(
     assert attack_primary_input3 != 0
     assert attack_primary_input4 != 0
 
-    print("proof_json => ", proof_json)
-    print("proof_json[inputs][3] => ", proof_json["inputs"][3])
-    print("proof_json[inputs][4] => ", proof_json["inputs"][4])
-    proof_json["inputs"][3] = hex(attack_primary_input3)
-    proof_json["inputs"][4] = hex(attack_primary_input4)
+    print("proof => ", proof)
+    print("proof[inputs][3] => ", proof.inputs[3])
```

Comment:
Why not: `print("proof.inputs[3] = ", proof.inputs[3])`? (same for the line below)

Reply:
The original code predates me, so not sure why it was `=>` instead of `=`.  Changed to `=`.

Reply:
Oh yes, there were two things in my comment above:
- `=>` -> `=`
- `proof.[inputs][3]` -> `proof.inputs[3]`

Reply:
updated